.github/workflows/tests.yml

```diff
@@ -30,26 +31,31 @@ jobs:
           echo "/home/linuxbrew/.linuxbrew/bin:/usr/bin:/bin" >> ${GITHUB_PATH}
         fi
 
-    - name: Test uninstalling GitHub Actions Homebrew
+    - name: Set up WSL
+      if: runner.os == 'windows'
+      uses: Vampire/setup-wsl@v1
```

Comment:
Given this is third-party, this should be pinned to a checksum.

Reply:
@reitermarkus Done. Is that why CI isn't even starting?

Reply:
@MikeMcQuaid `vampire/setup-wsl@bb4708da2512aac2a4c2947effc1061ecc0a9aef is not allowed to be used in Homebrew/install. Actions in this workflow must be: created by GitHub, within a repository owned by Homebrew or match the following: Homebrew/*, n1hility/cancel-previous-runs@*, peter-evans/create-or-updat...`

see https://github.blog/2020-10-05-fine-tune-access-to-external-actions/

Reply:
Yes, needs to be allowed first.

Reply:
> @MikeMcQuaid `vampire/setup-wsl@bb4708da2512aac2a4c2947effc1061ecc0a9aef is not allowed to be used in Homebrew/install. Actions in this workflow must be: created by GitHub, within a repository owned by Homebrew or match the following: Homebrew/*, n1hility/cancel-previous-runs@*, peter-evans/create-or-updat...`

Thanks! Where are you seeing that message?

Reply:
https://github.com/Homebrew/install/actions/runs/384968078